back/engines/free/user_confirmation/config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,15 @@
+class Rack::Attack
```

Comment:
Perhaps it's a stupid question, but will this work well in the sense that it won't overwrite the throttling in the core app? (because to me it looks like the `Rack::Attack` class gets defined twice and therefore only one of both might be used).

Reply:
Ah very good point, it should be extended instead 👍